src/core/collector.ml

```diff
@@ -53,10 +53,15 @@ module type S = sig
   (** Exit an explicit span.
       @since 0.3 *)
 
-  val add_data_to_span : span -> (string * user_data) list -> unit
-  (** @since Adds data to the given span.
+  val add_data : (string * user_data) list -> unit
+  (** @since Adds data to the ambient span.
```

Comment:
Since this library doesn't explicitly depend on `ambient-context`, the word I'd lean towards is "implicit span" — _how_ that's implicit is slightly up to the collector.
```suggestion
  val add_data : (string * user_data) list -> unit
  (** @since Adds data to the current, implicit span.
```